defs/build.bzl

```diff
@@ -0,0 +1,43 @@
+def _gcs_upload_impl(ctx):
+  targets = []
```

Comment:
skylark style guide says 2 space for regular indentation, 4 spaces for hanging indentation